experimental/PropertyDDS/packages/property-proxy/.eslintrc.js

```diff
@@ -3,17 +3,20 @@
  * Licensed under the MIT License.
  */
 
-const eslintConfig = require("@fluidframework/eslint-config-fluid");
+module.exports = {
+    "extends": [
+        "@fluidframework/eslint-config-fluid/eslint7"
+    ],
+    "parserOptions": {
+        "project": ["./tsconfig.json", "./src/test/tsconfig.json"]
+    },
+    "rules": {
+        // TODO(marcus): remove the linting issues
```

Comment:
I changed the eslint configuration to the same way other packages define their linting, I had to deactivate some of the linting rules for now since fixing them in this PR will be a rather large change that is probably better to be done in a separate PR